### PR TITLE
Bugfix for iodaconv path issues

### LINKED ISSUE #377: Fix 'lib_python' issue in ioda-converters PYTHONPATH for GDASApp ctests
@jiaruidong2017 's test now fails because ioda-converters now wants `import lib_python` rather than `pyiodaconv`... this needs to be dealt with in a generic way.

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
This PR closes #377 in that it solves the issue for the ims_snowcover test (others may need this later).

The JCSDA changed ioda-converters recently where it installs with `pip install` but it was done in a way in which all of the `ioda-converters/src` directories became python packages. This is inconsistent with the way the ioda-bundle installs the python scripts. So instead of `build/lib/pyiodaconv`, some scripts now need `iodaconv/src` on their `$PYTHONPATH`. This should probably be eventually resolved with JCSDA core staff, but this fix gets the test passing again.

### CODE REVIEW

**Review by RussTreadon-NOAA — COMMENTED** [review]
Change is straightforward.    I'll approve pending passes from the Hera and Orion GW-RT tests.

Question:  Is the JCSDA core team aware of this problem and have they opened an issue to address it?   We want to avoid patches becoming solutions.

**Comment by CoryMartin-NOAA** [comment]
@RussTreadon-NOAA I've had a back and forth with them and I thought this was 'solved' but I am not thrilled with the solution. I will open an issue this afternoon after a quick query of the open issues.

**Review by guillaumevernieres — APPROVED** [review]
Looks amazing @CoryMartin-NOAA .

**Comment by CoryMartin-NOAA** [comment]
see https://github.com/JCSDA-internal/ioda-converters/issues/1187

**Review by jiaruidong2017 — APPROVED** [review]
@CoryMartin-NOAA Thanks for your efforts.

**Comment by RussTreadon-NOAA** [comment]
Are the `test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY` failures expected?

**Comment by CoryMartin-NOAA** [comment]
@RussTreadon-NOAA I wasn't expecting them to fail... @guillaumevernieres any ideas?

**Comment by CoryMartin-NOAA** [comment]
The issue is unrelated to this PR and is already documented here https://github.com/JCSDA-internal/eva/issues/77

**Comment by RussTreadon-NOAA** [comment]
Got it.  Thank you @CoryMartin-NOAA , for identifying why `test_gdasapp_soca_JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY` fails.

**Review by RussTreadon-NOAA — APPROVED** [review]
Approve following successfully execution of ctests on Orion and Hera.

**Comment by guillaumevernieres** [comment]
> @RussTreadon-NOAA I wasn't expecting them to fail... @guillaumevernieres any ideas?

I'm looking at it right now @CoryMartin-NOAA .

**Comment by CoryMartin-NOAA** [comment]
This is passing now on orion, the fix is in EVA now as an open PR, I will merge now (can revert later if anyone has an issue with this).

**Pull request merged**
